samples/README_Windows.md

```diff
@@ -110,3 +110,8 @@ The following samples demonstrate how to develop enclave applications using OE A
 - Demonstrate how to configure an enclave to enable switchless calls originated within it
 - Recommend the number of host worker threads required for switchless calls in practice
 - Demonstrate how to enable switchless calls in an enclave application
+
+#### [Host Verification](host_verify/README.md)
```

Comment:
Suggest "Host-side Enclave Verification" since it doesn't verify hosts per se
```suggestion
#### [Host-side Enclave Verification](host_verify/README.md)
```